src/components/DayPickerRangeController.jsx

```diff
@@ -258,6 +258,11 @@ export default class DayPickerRangeController extends React.Component {
     if (didStartDateChange) {
       modifiers = this.deleteModifier(modifiers, this.props.startDate, 'selected-start');
       modifiers = this.addModifier(modifiers, startDate, 'selected-start');
+      if (this.props.startDate) {
+        const startSpan = this.props.startDate.clone().add(1, 'day');
```

Comment:
I wonder if we should do something like
```javascript
const {
  startDate: prevStartDate,
  minimumNights: prevMinimumNights,
  ...
} = this.props
```
so that we're not referencing `this.props` so many times in this function